webapp/graphite/render/evaluator.py

```diff
@@ -78,6 +78,27 @@ def evaluateTokens(requestContext, tokens, replacements=None):
     raise ValueError("unknown token in target evaluator")
 
 
+def extractPathExpressions(targets):
```

Comment:
Could you add a small unittest for this one ? (There is already a quite similar function in functions.py with some unit tests)